README.md

```diff
@@ -206,6 +220,8 @@ docker run --name phpmyadmin -d -e PMA_PASSWORD_FILE=/run/secrets/db_password.tx
 - `PMA_CONTROLHOST`
 - `PMA_CONTROLUSER`
 - `PMA_CONTROLPASS`
+- `PMA_SSL`
+- `PMA_SSLS`
```

Comment:
@TasdidurRahman I am considering a revert of the two lines
It makes no sense to read 1 and 0 from a file
What do you think ?

Reply:
Reverted as 577a94fda60fbd1bb13247531c1783db9c51a64b

Reply:
sorry for late reply, LGTM

Reply:
Thank you